build/gen-codes-v2-validate/go.mod

```diff
@@ -0,0 +1,3 @@
+module validate
+
+go 1.18
```

Comment:
Maybe `go.mod` is not required here as a separate project. However, in that case main codebase will be needed to bumped to 1.18 version due to usage of generics.  The benefit of that change would be skipping `codes.go` copy requirement, but as a consequence validator will be not able to run under windows OS (unless there is some workaround for unsupported GOOS that I'm not aware of)

Reply:
Sure, additional `go.mod` removed then, as far as you're okay with `go-evdev` being bound to go >=1.18 version